src/handle/root.rs

```diff
@@ -43,6 +43,21 @@ impl<T: Object> Root<T> {
     /// The caller _must_ ensure `Root::into_inner` or `Root::drop` is called
     /// to properly dispose of the `Root<T>`. If the value is dropped without
     /// calling one of these methods, it will *panic*.
+    ///
+    /// Be careful that you aren't short-circuiting with an early return before
```

Comment:
Since this communicates the same as above, can it be reworded to present it as an example?

> For example, early return with a `?` may cause a function to return before freeing a `Root`. ...

Reply:
I've attempted a rewording along the lines suggested here.